skimage/segmentation/random_walker_segmentation.py

```diff
@@ -428,8 +428,8 @@ def random_walker(data, labels, beta=130, mode='cg_j', tol=1.e-3, copy=True,
     # Parse input data
     if mode not in ('cg_mg', 'cg', 'bf', 'cg_j', None):
         raise ValueError(
-            "{mode} is not a valid mode. Valid modes are 'cg_mg',"
-            " 'cg', 'cg_j', 'bf' and None".format(mode=mode))
+            f"{mode} is not a valid mode. Valid modes are 'cg_mg',"
+            " 'cg', 'cg_j', 'bf' and None")
```

Comment:
```suggestion
            f"{mode} is not a valid mode. Valid modes are 'cg_mg', "
            f"'cg', 'cg_j', 'bf', and None")
```
out of consistency and as per this discussion https://github.com/scikit-image/scikit-image/pull/5478#discussion_r671584470